doc/examples/effects/simple/lib/main_move.dart

```diff
@@ -17,9 +18,19 @@ class MyGame extends BaseGame with TapDetector {
   @override
   void onTapUp(details) {
     square.addEffect(MoveEffect(
-      destination: details.localPosition.toVector2(),
+      path: [
+        details.localPosition.toVector2().clone(),
```

Comment:
toVector creates a new clone, so clone is unnecessary. cc @erickzanardo this is how you do it.

Reply:
Fixed